domains/razin.json

```diff
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "Razin Rayees",
+    "email": "razin.rayeesk@gmail.com"
+  },
+
+  "record": {
+    "CNAME": "razinrayees.netlify.app"
```

Comment:
We do not support Netlify. You need to read the requirement boxes on your PRs which you have checked this.